perfkitbenchmarker/linux_benchmarks/cloudsuite_data_serving_benchmark.py

```diff
@@ -212,10 +210,6 @@ def CleanupServer(vm):
     vm.RemoteCommand('sudo docker stop cassandra-server')
     CleanupServerCommon(vm)
 
-  def CleanupClient(vm):
-    vm.RemoteCommand('sudo docker rmi cloudsuite/data-serving:client')
```

Comment:
Not when docker is removed, but when the package installed with vm.Install('cloudsuite/data-serving:client') is automatically uninstalled by PKB. That is, during teardown, PKB uninstalls packages that were installed with vm.Install(X) via vm.Uninstall(x). Caveat: PKB only bothers with the uninstall on static VMs; other VMs are deleted, so there's no point in uninstalling.